src/packages.builds

```diff
@@ -15,6 +16,16 @@
     </Project>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(BuildAllPackages)' == 'false' AND '$(SkipManagedPackageBuild)' != 'true'" >
+    <Project Include="$(MSBuildThisFileDirectory)..\pkg\Microsoft.Private.PackageBaseline\Microsoft.Private.PackageBaseline.builds">
+      <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
+    </Project>
+    <!-- add specific builds / pkgproj's here to include in servicing builds -->
```

Comment:
I think the only packages we want to build by default are the ones with Private in their name.  See https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/2.0.0/Latest_Packages.txt. So you can remove the Microsoft.NETCore.Targets and add the other Private root packages (you don't need to explicitly list out the RID versions of those as they are covered in the builds file).